notebooks/official/workbench/subscriber_churn_prediction/telecom-subscriber-churn-prediction.ipynb

```diff
@@ -33,16 +33,17 @@
         "\n",
```

Comment:
<p>combine installs into a single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
please find that all installs are in a single cell

Reply:
<p>The install should be moved into the same code block above (i.e., where USER_FLAG is set)</p>

Reply:
done